apiserver/DEVELOPMENT.md

```diff
@@ -4,70 +4,130 @@ This guide covers the purpose, requirements, and deployment of the Kuberay API S
 
 ## Requirements
 
-| Software | Version  |                                                            Link |
-| :------- | :------: | ------------------------------------------------------------: |
+| Software | Version  |                                                                Link |
+| :------- | :------: | ------------------------------------------------------------------: |
 | kubectl  | v1.18.3+ | [Download](https://kubernetes.io/docs/tasks/tools/install-kubectl/) |
-| Go       |  v1.13+  |                            [Download](https://golang.org/dl/) |
-| Docker   |  19.03+  |                  [Download](https://docs.docker.com/install/) |
+| Go       |  v1.17   |                                  [Download](https://golang.org/dl/) |
```

Comment:
@blublinsky The go version does not match the minimal support go version in the [go.mod ](https://github.com/ray-project/kuberay/blob/2b8947ccec2eccf528b0e1d17c72241d441e9c54/apiserver/go.mod#L3)file.

Reply:
@blublinsky No need to do anything. @kevin85421 created the [issue](https://github.com/ray-project/kuberay/issues/1329) to update the go version.